test/tests/ver-1a/comparison_ver-1a.py

```diff
@@ -5,30 +5,239 @@
 import pandas as pd
 
 #===============================================================================
+# Physical constants
+kb = 1.380649e-23  # J/K Boltzmann constant
+R = 8.31446261815324 # J/mol/K Gas constant
 
-fig = plt.figure(figsize=[6.5,5.5])
-gs = gridspec.GridSpec(1,1)
-ax = fig.add_subplot(gs[0])
+def get_roots_TMAP4(L, alpha_max, step=0.0001):
+    # https://stackoverflow.com/questions/28766692/how-to-find-the-intersection-of-two-graphs/28766902#28766902
+    """Gets the roots of alpha = L / tan(alpha)
+
+    Args:
+        L (float): parameter L
+        alpha_max (float): the maximum alpha to consider
+        step (float, optional): the step discretizing alphas.
+            The smaller the step, the more accurate the roots.
+            Defaults to 0.0001.
+
+    Returns:
+        np.array: array of roots
+    """
+    alphas = np.arange(0, alpha_max, step=step)[1:]
+    # Define graph as alphas
+    f = alphas
+    # Define graph as L / tan(alphas)
+    g = L / np.tan(alphas)
+    # Find intersections of the two graphs
+    idx = np.argwhere(np.diff(np.sign(f - g))).flatten()
+    # remove one every other idx
+    idx = idx[::2]
+    roots = alphas[idx]
+    return roots
+
+# taken from https://mooseframework.inl.gov/TMAP8/verification/ver-1a.html#longhurst1992verification
+def analytical_expression_fractional_release_TMAP4(t,P_0, D, S, V, T, A, l):
+    """
+    Analytical expression for the fractional release given by TMAP4 report
+
```

Comment:
```suggestion
def analytical_expression_fractional_release_TMAP4(t,P_0, D, S, V, T, A, l):
    """
    Analytical expression for the fractional release given by TMAP4 report

    Taken from the TMAP4 V&V Report (https://doi.org/10.2172/10174725)
    
```

Let's make this more consistent with the TMAP7 method description, and use an OSTI DOI for the link - the one to the verification documentation is decidedly unclear about what portion of the page it is referring to, when it just drops you to the bottom of the page.